### PR TITLE
[NeoDays] Monster cleaning and spider resizing

### PR DESCRIPTION
**Pull request by Golfavel**
#### Summary
Cleans multiple subdirectories of monsters, makes spider sizes consistent throughout evolutions

#### Content of the change
Cleans directories:
 - ``migo/*``
 - ``insect/*``
 - ``fungus/*``
 - ``blob/*``
 - ``etc/*``
 - ``triffid/*``

Many sprites for creatures in base DDA located some depth in ``monsters/mod/*`` were moved to appropriate directory under ``monsters/*/``
Spiders worked on to have more consistent sizing with their respective evolution
Fawns now use small deer sprite previously in ``monsters/mod/unknown/``

#### Testing
Composer threw no errors, sprites appeared as before/intended
_Spider resizing. Small cellar spider has been fixed since this image was captured_
neo_spid
_Fawns next to an adult deer and the missing small cellar spider_
neo_smold

#### Additional information
I kept the json files for most mod creatures not tagged with ``_pk``.

### CODE REVIEW

File: gfx/NeoDays/pngs_tiles_10x10/monsters/feral_humans/mon_shia.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "mon_shia",
+    "mon_zombie_jackson"
```
Comment (Fris0uman) [inline gfx/NeoDays/pngs_tiles_10x10/monsters/feral_humans/mon_shia.json:4]: those two are crazy cataclysm monsters

File: gfx/NeoDays/pngs_tiles_10x10/monsters/triffid/mon_creeper_vine.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "id": "mon_creeper_vine",
+  "fg": "mon_creeper_vine_0",
+  "rotates": false,
+  "//": "below doesnt work (as intended? it uses '_2')",
+  "multitile": true,
```
Comment (Fris0uman) [inline gfx/NeoDays/pngs_tiles_10x10/monsters/triffid/mon_creeper_vine.json:6]: I don't think monsters can use multitile like this

Comment (Golfavel) [reply gfx/NeoDays/pngs_tiles_10x10/monsters/triffid/mon_creeper_vine.json:6]: I'll move them to scratch.

File: gfx/NeoDays/pngs_tiles_10x10/monsters/feral_humans/mon_shia.json (hunk 1 above)
Comment (Golfavel) [reply gfx/NeoDays/pngs_tiles_10x10/monsters/feral_humans/mon_shia.json:4]: They're both in standard DDA; but I'll shift things.

**Pull request merged**